autoparams/src/main/java/org/javaunit/autoparams/customization/PropertyWriter.java

```diff
@@ -0,0 +1,52 @@
+package org.javaunit.autoparams.customization;
+
+import java.beans.Introspector;
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import org.javaunit.autoparams.generator.ObjectContainer;
+import org.javaunit.autoparams.generator.ObjectGenerationContext;
+import org.javaunit.autoparams.generator.ObjectGenerator;
+import org.javaunit.autoparams.generator.ObjectQuery;
+
+public final class PropertyWriter implements Customizer {
+
+    @Override
+    public ObjectGenerator customize(ObjectGenerator generator) {
+        return new PropertyWriterPostProcessor(generator);
```

Comment:
이렇게 하면 내부 클래스를 안만들어도 될 것 같은데 구현의 문제고 취향 차이일 수 있으니 중요한 건 아니고 한 번 봐주세요.

```suggestion
        return (query, context) -> {
            ObjectContainer container = generator.generate(query, context);
            writeProperty(container.unwrapOrElseThrow(), context);
            return container;
        };
```

Reply:
@gyuwon 
inner class 이름 고민 안해서 더 좋네요
리뷰 반영해서 수정했습니다.